### PR TITLE
Design tokens: Add font tokens for size, weight, and family

### PR DESCRIPTION
**Pull request by ayeshakmaz**
#### What changed?
Add tokens for font-size, font-family, and font-weight + documentation
(Also updates our roboflow 100 to #F1F1F1)

#### Why?
Create tokens for all platforms to align on typography decisions 

### Links

- [Jira](https://jira.pinadmin.com/browse/GESTALT-3006)

### Checklist

- [X] Added documentation + accessibility tests
- [X] Checked stakeholder feedback (e.g. Gestalt designers)

### CODE REVIEW

**Comment by ponori** [comment]
This is looking _so good_!

I'm coming in hard with the nittiest of nits - so just feel free to ignore as it's not critical for this work. Would there be any way to align the table columns for all tables so that the content flows more predictably from one token family to the next?

**Comment by ayeshakmaz** [comment]
@ponori I totally agree that would be ideal, and actually tried for a good chunk of time to get them aligned, but I think it will involve work on our actual Table component, so I can create a ticket for that.

**Comment by ponori** [comment]
Totally understandable and yeah, that seems like a solid next step. Thanks!

File: packages/gestalt-design-tokens/tokens/color/base.json [hunk 1]
```diff
@@ -136,7 +136,7 @@
     "gray": {
       "roboflow": {
         "50": { "value": "#F9F9F9" },
-        "100": { "value": "#F3F1F1" },
+        "100": { "value": "#F1F1F1" },
```
Comment (dangerismycat) [inline packages/gestalt-design-tokens/tokens/color/base.json:139]: 🏎

File: packages/gestalt-design-tokens/tokens/font/base.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          "comment": "Default font family used at Pinterest"
+        },
+        "japanese": {
+          "value": "'SF Pro JP', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Fira Sans', 'Droid Sans', 'Helvetica Neue', Helvetica, 'ヒラギノ角ゴ Pro W3', 'Hiragino Kaku Gothic Pro', 'メイリオ', Meiryo, 'ＭＳ Ｐゴシック', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'",
+          "comment": "Font family for Japanese"
+        }
+      },
+      "code": {
+        "value": "SFMono-Medium, 'SF Mono', 'Segoe UI Mono', 'Roboto Mono', 'Ubuntu Mono', Menlo, Consolas, Courier, monospace",
+        "comment": "Font family for showing things like code or computer values"
```
Comment (dangerismycat) [inline packages/gestalt-design-tokens/tokens/font/base.json:46]: didn't know we had a recommendation for this — neat!

**Review by dangerismycat — APPROVED** [review]

File: packages/gestalt-design-tokens/tokens/color/base.json (hunk 1 above)
Comment (ayeshakmaz) [reply packages/gestalt-design-tokens/tokens/color/base.json:139]: 💯

**Pull request merged**